api/src/main/java/com/datastrato/gravitino/authorization/Privilege.java

```diff
@@ -21,7 +21,67 @@ public interface Privilege {
 
   /** The name of this privilege. */
   enum Name {
-    /** The privilege of load a catalog. */
-    LOAD_CATALOG
+    /** The privilege to show catalogs. */
+    SHOW_CATALOG,
+    /** The privilege to use a catalog. */
+    USE_CATALOG,
+    /** The privilege to create a catalog. */
+    CREATE_CATALOG,
+    /** The privilege to alter a catalog. */
+    ALTER_CATALOG,
+    /** The privilege to drop a catalog. */
+    DROP_CATALOG,
+    /** The privilege to show schemas. */
+    SHOW_SCHEMA,
+    /** The privilege to use a schema. */
+    USE_SCHEMA,
+    /** The privilege to create a schema. */
+    CREATE_SCHEMA,
+    /** The privilege to alter a schema. */
+    ALTER_SCHEMA,
+    /** The privilege to drop a schema. */
+    DROP_SCHEMA,
+    /** The privilege to show tables. */
+    SHOW_TABLE,
```

Comment:
SHOW_TABLES or SHOW_TABLE, I think for SHOW_XXX, it should add "s", right?

Reply:
From UI's view, to show resources tree. Maybe we don't need show metalakes, show tables, show schemas privilege. Every one should have the privilege.

Reply:
Em...OK, I will correct this. My concern is that we don't use LOAD_A_SCHEMA.  But I really want to remove this.